### PR TITLE
libsbml 5.11.0 (new formula)

### PR DESCRIPTION
**Pull request by mythosil**
LibSBML is a free, open-source programming library
to help you read, write, manipulate, translate, and validate
SBML files and data streams.

### CODE REVIEW

**Comment by mythosil** [comment]
the outage of sourceforge made the build failed..

**Comment by iml** [comment]
```
==> audit problems
libsbml:
 * Stable: version 5.11.0 is redundant with version scanned from URL
 * Use http://downloads.sourceforge.net to get geolocation (url is http://sourceforge.net/projects/sbml/files/libsbml/5.11.0/stable/libSBML-5.11.0-core-plus-packages-src.tar.gz).
 * `require 'formula'` is now unnecessary
 * 33: Trailing whitespace was found

File: libsbml.rb [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  LANGUAGES_OPTIONAL.each do |opt, lang|
+    option "with-#{opt}", "generate #{lang} interface library [default=no]"
+  end
+
+  depends_on 'cmake' => :build
+  depends_on 'swig' => :build
+  depends_on :python => :optional
+
+  # fix ruby's sitelib dir
+  def patches; DATA; end
```
Comment (iml) [inline libsbml.rb]: `patch :DATA`

File: libsbml.rb [hunk 2]
```diff
@@ -0,0 +1,78 @@
+require 'formula'
```
Comment (dpo) [inline libsbml.rb]: This is no longer necessary.

File: libsbml.rb [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+class Libsbml < Formula
+  homepage 'http://sbml.org/Software/libSBML'
+  url 'http://sourceforge.net/projects/sbml/files/libsbml/5.11.0/stable/libSBML-5.11.0-core-plus-packages-src.tar.gz'
+  version '5.11.0'
+  sha1 'cfe9697bf9e28758c0dbd7b7c6b6ad3a7a01c56e'
```
Comment (dpo) [inline libsbml.rb]: Please use double quotes instead of single quotes. `brew audit libsbml --strict` will produce a full report.

**Comment by mythosil** [comment]
Should I squash the commits?

**Comment by dpo** [comment]
Yes please.

**Comment by dpo** [comment]
@BrewTestBot test this please

**Comment by mythosil** [comment]
I don't know why `brew doctor` fails. Is there any way to fix the issue?

**Comment by dpo** [comment]
No need to worry about the TravisCI build. It's not quite reliable yet.

**Comment by dpo** [comment]
There seems to be a problem with this PR. I can't pull it. Would you mind closing this and opening a new PR with the latest version of your formula?

There are still two minor issues to address:
* line 3 should be replaced with the equivalent `downloads.sourceforge.net` url
* line 23: there was no need to replace *that* single quote ;)

**Pull request closed without merging**